doc/source/ray-core/fault-tolerance.rst

```diff
@@ -0,0 +1,63 @@
+Fault tolerance
+===============
+
+Ray is a distributed system, and that means failures can happen. Generally, failures can
+be classified into two classes: 1) failures internal to Ray, and 2) application-level failures. The first can be triggered by node failures,
+network failures, or just bugs in Ray. The latter can happen because of bugs in user-level code, or if external systems fail.
+Here, we describe the mechanisms that Ray provides to allow applications to recover from failures.
+
+.. _fault-tolerance_ownership:
+
+Ownership model
+---------------
+
+To understand the failure model of Ray, we need to understand the ownership
+model of Ray first. The owner of an object or non-detached actor is defined as
+the worker who creates it, and the detached actor’s owner is always GCS.
+Consider the following example: 
+
+.. code-block:: python
+    
+    @ray.remote
+    class Actor:
+        pass
+
+
+    @ray.remote
+    def foo():
+        obj = ray.put(10)
+        actor = Actor.remote()
+        actor_detached = Actor.options(lifetime=”detached”).remote()
+        return (obj, actor, actor_detached)
+
+    # the driver owns obj1 since the driver created it
+    obj1 = ray.put(10)
+
+    # obj2 is owned by the driver since it’s also the driver who created it. 
+    obj2 = foo.remote()
+
+    # obj3 and actor1 are owned by the worker where foo.remote() is executed.
+    # actor2 is owned by the worker since it’s a detached actor.
+    (obj3, actor1, actor2) = ray.get(obj2)
```

Comment:
Isn't actor2 owned by the GCS since it's a detached actor since that what you describe above. Detached actors are always owned by GCS.